pennylane/drawer/draw.py

```diff
@@ -18,20 +18,17 @@
 """
 from functools import wraps
 import warnings
-import pkg_resources
 
 import pennylane as qml
 from .tape_mpl import tape_mpl
 from .tape_text import tape_text
 
 
 def catalyst_qjit(qnode):
-    """The ``catalyst.while`` wrapper method"""
-    try:
-        pkg_resources.get_distribution("pennylane_catalyst")
+    """Check if Catalyst is available and if the QNode is qjitted."""
+    if "catalyst" in qml.compiler.available_compilers():
         return qnode.__class__.__name__ == "QJIT"

```

Comment:
I know this came from another PR, but can't we just return this conditional? when would there be no available compiler, but someone changed the QNode class name to "QJIT"?

Reply:
Here it is necessary because you are outside the QJIT context. You can not call the drawer inside the context at least for the moment.

Reply:
to be clear, I meant to just always return `qnode.__class__.__name__ == "QJIT"`, not `qml.compiler.available("catalyst")`

Reply:
Solved offline: we will keep it just in case.